daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala

```diff
@@ -475,17 +474,17 @@ class OrderingSpec
     new util.ArrayList[X](as.asJava)
 
   private val txSeed = crypto.Hash.hashPrivateKey("SBuiltinTest")
-  private def dummyMachine = Speedy.Machine fromExpr (
-    expr = e"NA:na ()",
+  private def initMachine(expr: SExpr) = Speedy.Machine fromSExpr (
+    sexpr = expr,
     compiledPackages = PureCompiledPackages(Map.empty, Map.empty),
-    scenario = false,
     Time.Timestamp.now(),
-    Some(txSeed),
+    InitialSeeding(Some(txSeed)),
+    Set.empty,
   )
 
   private def translatePrimValue(v: Value[Value.AbsoluteContractId]) = {
-    val machine = dummyMachine
-    machine.ctrl = SEImportValue(v)
+    val expr = SEImportValue(v)
+    val machine = initMachine(expr)
```

Comment:
Let's keep it concise. 😃
```suggestion
    val machine = initMachine(SEImportValue(v))
```